index.html

```diff
@@ -146,6 +146,40 @@
     </p>
   </main>
 
+  <div class="footer">
+    <a class="footerlogo" href="https://ekaterinburg.design/">
```

Comment:
What about camelCase for CSS classes?

`footerLogo__image` :)

Reply:
done